Alooma-iOS/Alooma.m

```diff
@@ -116,9 +121,17 @@ - (instancetype)initWithToken:(NSString *)apiToken serverURL:(NSString *)url
         self.timedEvents = [NSMutableDictionary dictionary];
         _application = application;
 
-        if (self.application) {
-          [self setUpListeners];
+        if (![Alooma isAppExtension]) {
+            if ([[UIApplication class] respondsToSelector:@selector(sharedApplication)]) {
+                _application = [[UIApplication class] performSelector:@selector(sharedApplication)];
+            }
+#if !defined(ALOOMA_APP_EXTENSION)
```

Comment:
Should this be removed? same for the other one in this file.

And can we remove our addition of the nullable `application`?

Reply:
Keeping this as we don't want to diverge too much from upstream now that the issue is fixed.

I changed the PR to keep the diff from upstream to a minimum. Note that they added a nullable `UIApplication` property.

I suggest to manually look at the diff from `upstream/master`.